### PR TITLE
SUP-1560 Add bk build new

### PR DESCRIPTION
**Pull request by lizrabuya**
- Add `bk build new` to  create build with a pipeline name as argument
- Add tests for parsing pipeline args

### CODE REVIEW

**Review by james2791 — DISMISSED** [review]
🚅

File: pkg/cmd/build/new.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	var message string
+	var commit string
+	var branch string
+	var web bool
+
+	cmd := cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "new <pipeline> [flags]",
+		Short:                 "Creates a new pipeline build",
+		Args:                  cobra.ArbitraryArgs,
```
Comment (jradtilbrook) [inline pkg/cmd/build/new.go:26]: I think this should be:
```suggestion
		Args:                  cobra.ExactArgs(1),
```

File: pkg/cmd/build/new.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		Long: heredoc.Doc(`
+			Creates a new build for the specified pipeline and output the URL to the build.
+
+			It accepts {pipeline_slug}, {org_slug}/{pipeline_slug} or a full URL to the pipeline as an argument.
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println("build new command args here: ", args)
```
Comment (jradtilbrook) [inline pkg/cmd/build/new.go:33]: Looks like debugging left in?

File: pkg/cmd/build/new.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+			if len(args) == 1 {
```
Comment (jradtilbrook) [inline pkg/cmd/build/new.go:34]: You won't need this with the change to line 26 above

**Review by jradtilbrook — COMMENTED** [review]
Looks good. Just a few things

File: pkg/cmd/build/new.go (hunk 1 above)
Comment (lizrabuya) [reply pkg/cmd/build/new.go:26]: I'll make this change together with the suggestions below.

**Review by jradtilbrook — APPROVED** [review]

**Pull request merged**